src/app/page.tsx

```diff
@@ -1,7 +1,8 @@
+import Navigation from "@/components/navigation";
 const Home = () => {
   return (
-    <div className="flex h-screen w-screen items-center justify-center">
-      Hello World!!
+    <div className="flex h-screen w-screen justify-center space-x-2">
```

Comment:
also remove h-screen and w-screen it'll mess up the width of your nav bar here
```suggestion
    <div>
```